frontend/src/views/Home.vue

```diff
@@ -36,27 +36,11 @@
       </p>
     </div>
 
-    <v-container ref="content-start" style="padding: 20px 30px 10px 30px">
-      <HeaderTitle :title="'upcoming events'" />
-      <v-row>
-          <EventGrid :events="eventItems" v-if="time < 60 * 1440 * 60000"></EventGrid>
-          <!-- <div
-            class="fb-page"
-            data-href="https://www.facebook.com/csesoc"
-            data-tabs="events"
-            data-small-header="false"
-            data-adapt-container-width="true"
-            data-hide-cover="false"
-            data-show-facepile="false"
-          >
-            <blockquote cite="https://www.facebook.com/csesoc" class="fb-xfbml-parse-ignore">
-              <a href="https://www.facebook.com/csesoc">Loading...</a>
-            </blockquote>
-          </div> -->
-      </v-row>
+    <v-container ref="content-start">
+      <HeaderTitle :title="'upcoming events'"/>
+      <EventDisplay :events="eventItems" v-if="time < 60 * 1440 * 60000"></EventDisplay>
```

Comment:
Whats this for?

Reply:
like the v-if part.

Reply:
This was to prevent showing outdated data on the page, but I've refactored this code since for extra clarity, and to show a more helpful message to the user.